mozilla-lockbox/lockbox-android
# More reliably clear the clipboard

Fixes _TBD_

This patch re-uses the existing broadcast receiver and an alarm to clear the clipboard.  The set alarm is using the most power-friendly configuration; it will fire if the device is running in some form, will not wake it from sleep, should fire at next wakeup, and will not survive a reboot.

## Testing and Review Notes

0. Lockbox running in foreground, with an entry's details displayed
1. tap the copy for either Username or Password
2. send Lockbox to background
3. wait at least 60 seconds
4. attempt to paste

**Expected** nothing (technically the empty string is pasted)


## To Do

  - optional: consider adding instrumentation (integration/UI) tests
- consider running this branch in a debug simulator and check for memory leak notifications or any warnings
